skore/src/skore/sklearn/_estimator/metrics_accessor.py

```diff
@@ -70,23 +70,19 @@ def report_metrics(
     ) -> pd.DataFrame:
         """Report a set of metrics for our estimator.
 
-        Parameters
+                Parameters
```

Comment:
The indent shouldn't change
```suggestion
        Parameters
```